sekizai/templatetags/sekizai_tags.py

```diff
@@ -41,6 +41,18 @@ def import_processor(import_path):
     return getattr(module, object_name)
 
 
+def import_mapped_processors(sekizai_processors):
+    mapped_processors = {}
+    for name, processor in sekizai_processors.items():
+        mapped_processors[name] = import_processor(processor)
+    return mapped_processors
+
+_mapped_preprocessors = import_mapped_processors(
```

Comment:
Is there no way we can get rid of these at-import globals? Could this move inside the template tag classes (``__init__`` or ``render`` or somewhere)?